### DISCUSSION TITLE
Can't check DNS records for added domain

### DISCUSSION THREAD

**Discussion by khalidelmerrah** [postalserver/postal#1971] [Help with using Postal]
hello, we just finished installing Postal, and it can't check dns for added domain, the records are correctly set but postal takes long to check and then shows error that dns records are incorrect. 

i'm asking if there is an issue with our vps ( firewall blocking or something ) that could prevent this. 

thanks

**Comment by willpower232** [comment]
If you ssh onto the server, you can try `dig txt yourdomain.com` to see if your server is able to resolve DNS properly.
